content/dart/concepts/list/terms/join/join.md

```diff
@@ -0,0 +1,110 @@
+
+&nbsp;  
+
+
+## Example No 1
+
+
+
+The following example is showing a list with examples of different separators (only particular examples amongst many variations) and their respective outputs:
+
+```dart
+
+List<String> chores = ["planting flowers", "vacuuming", "laundry", "ironing", "preparing meals"] // our main list that will  refer to
+
+
+/* "planOfDut" as a variable holding the modified list ("chores") by the method ".join()" */
+
+String planOfDut = chores.join(); // removes whitespaces between elements
+print(planOfDut); // returns:  planting flowers,vacuuming,laundry,ironing,preparing meals
+
+String planOfDut = chores.join(""); // no separation whatsoever
+print(planOfDut); // returns:  planting flowersvacuuminglaundryironingprepering meals
+
+String planOfDut = chores.join(" / "); // adds a slash
+print(planOfDut); // returns:  planting flowers / vacuuming / laundry / ironing / preparing meals
+
+String planOfDut = chores.join(" * "); // adds an asterisk
+print(planOfDut); // returns:  planting flowers * vacuuming * laundry * ironing * preparing meals
+
+String planOfDut = chores.join(" or "); // adds 'or' as a conjunction (any conjunction can be chosen)
+print(planOfDut); // returns:  planting flowers or vacuuming or laundry or ironing or preparing meals
+
+String planOfDut = chores.join(" and again "); // adds 'and again'
+print(planOfDut); // returns:  planting flowers and again vacuuming and again laundry and again ironing and again preparing meals
+
+String planOfDut = chores.join(" and maybe tomorrow "); // adds 'and again'
+print(planOfDut); // returns:  planting flowers and maybe tomorrow vacuuming and maybe tomorrow laundry and maybe tomorrow ironing and maybe tomorrow preparing meals
+
+String planOfDut = chores.join(" (sigh) "); // removes whitespaces between elements
+print(planOfDut); // returns:  planting flowers (sigh) vacuuming (sigh) laundry (sigh) ironing (sigh) preparing meals
+
+
+```
+&nbsp;  
+
+
+## Example No 2
+
+
+
+The below snippet is showing another list but combined with a predefined text to make it slightly more complex, plus the output:
+
+```dart
+
+List<String> resolutions = ["takeaways", "alcohol", "toxic people", "overtime at work", "binge watching", "cigarettes", "being a couch potato", "comparing to others", "all night parties"]
+
+String wishList = resolutions.join(", No ");
+
+print('In 2024 defenitely I will stick to these simple rules: No ${wishList}.'); // returns: In 2024 defenitely I will stick to these simple rules: No takeaways, No alcohol, No toxic people, No overtime at work, No binge watching, No cigarettes, No being a couch potato, No comparing to others, No all night parties.
+
+
+```
+
+&nbsp;  
+
+
+### Tags
+
+```
+Concatenation
+Dart
+Lists
+
+```

```

Comment:
Tags should not be included here. Please follow this [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md).